<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/images/pokerLogo.png</normaloff>:/images/pokerLogo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-image: url(:/images/background.png);	
	background-position: center;
}

QPushButton:enabled{
	background-color: rgb(85, 88, 93, 90);
    border: 2px solid #8f8f91;
    border-radius: 6px;
    min-width: 80px;
}

QPushButton:hover {
	background-color: rgb(79, 186, 196, 150);
}

QPushButton:pressed {
	background-color: rgb(56, 47, 165);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Valid Table</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>158</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="contractButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="menuDaemonRPCinput">
     <property name="minimumSize">
      <size>
       <width>356</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string> Daemon RPC Address:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255, 45);</string>
     </property>
     <property name="text">
      <string>♣♣♣♣♣♣ Table v0.1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>421</width>
       <height>91</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/pokerLogo.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="contractLineEdit">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string> Table Contract Address:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Menu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
